linkerd/app/gateway/src/config.rs

```diff
@@ -51,7 +51,7 @@ impl Config {
                 Allow(self.allow_discovery),
             ))
             .check_new_service::<inbound::Target, http::Request<http::boxed::Payload>>()
-            .instrument(|_: &inbound::Target| info_span!("gateway"))
+            .instrument(|_: &inbound::Target| debug_span!("gateway"))
```

Comment:
IMO we might want to keep this one on at info, so we can tell what errors/warnings come from the gateway...

Reply:
I don't this is really practically a concern... You're either looking at a gateway pod or not.